src/environments/environment.ts

```diff
@@ -1,9 +1,11 @@
 // This file can be replaced during build by using the `fileReplacements` array.
 // `ng build --prod` replaces `environment.ts` with `environment.prod.ts`.
 // The list of file replacements can be found in `angular.json`.
+// isDebugMode - 'INFO' 'WARN 'ERROR'
 
 export const environment = {
```

Comment:
create interface: 

export interfaces  ENVIROMENT {
 production: boolean,
isDebugMode: 'INFO', 'WARN', 'ERROR'
}

export const enviroment: ENVIROMENT {

}

Reply:
this way we make sure the user does not accidentally put a wrong value in the property.